### ISSUE TITLE
Feature Request: Possibility to set list of locales

### ISSUE THREAD

**Issue by andzno1** [Neurrone/beets-audible#59]
I am struggling tagging my collection of German audio dramas. I got beets-audible working but it doesn't return any results for some of my German audio dramas. E.g.:
```
audible: Searching Audible for Der blinde Hellseher TKKG
Evaluating 0 candidates.
```

Entering that exact search term on the German audible page (audible.de) yields a result though:
![image](https://github.com/Neurrone/beets-audible/assets/4320938/1f0c9e1d-73bd-405c-a8bf-9d054b93909e)

while entering the same search term on the US page (audible.com) doesn't yield any result:
![image](https://github.com/Neurrone/beets-audible/assets/4320938/10ce3200-2a20-4b51-b7af-c4fc93f808c5)

Therefore it would be a really nice feature to be able to configure the plugin similar to beets' `import.languages` setting, e.g:
```yaml
audible:
  languages: de en
```
That new setting would enable us to provide a list of language locales making it possible to query different audible endpoints in order of the given list.

**Comment by peterpiglet** [comment]
this functionality would be awesome!

**Comment by Neurrone** [comment]
Better region support has been released with v1.0 of Beets-audible, which includes the changes from #67.

See the [Changelog](https://github.com/Neurrone/beets-audible/blob/main/CHANGELOG.md) for the list of changes.
